pisa_utils/models/data_models.py

```diff
@@ -0,0 +1,717 @@
+import logging
+from enum import Enum
+from typing import Literal, Optional
+
+from pydantic import BaseModel, Field, RootModel, field_validator
+
+from pisa_utils.models.labels import (
+    ATOM_LABEL,
+    ATOM_LABEL_EXAMPLES,
+    BOND_DISTANCES,
+    CHAIN_ID,
+    CHAIN_ID_EXAMPLES,
+    INTERFACE_AREA,
+    INTERFACE_CSS,
+    INTERFACE_H_BONDS,
+    INTERFACE_N_ATOMS,
+    INTERFACE_N_BONDS,
+    INTERFACE_N_RESIDUES,
+    INTERFACE_NUMBER,
+    INTERFACE_OTHER_BONDS,
+    INTERFACE_P_VALUE,
+    INTERFACE_SALT_BRIDGES,
+    INTERFACE_SOLVATION_ENERGY,
+    INTERFACE_SS_BONDS,
+    INTERFACE_TOTAL,
+    PDB_ID,
+    PISA_BURIED_SURFACE_AREAS,
+    PISA_STABILIZATION_ENERGY,
+    PRECISION_DP,
+    RESIDUE_3_LETTER_CODE,
+    RESIDUE_3_LETTER_EXAMPLES,
+    RESIDUE_SEQ_ID,
+    STANDARD_DP,
+    STATUS,
+    SYMMETRY_ID,
+)
+
+LOGGER = logging.getLogger(__name__)
+AS_IS_SYNONYM = "--as-is"
+
+
+def convert_single_obj_to_list(v, dtype=dict):
+    """Convert single dict to list if needed."""
+    if isinstance(v, dtype):
+        return [v]
+    return v
+
+
+class LigandPosition(Enum):
+    AUTO = "auto"
+    FIXED = "fixed"
+    FREE = "free"
+
+
+class Bond(BaseModel):
+    # First chain info
+    chain_1: str = Field(
+        ..., description=CHAIN_ID, examples=CHAIN_ID_EXAMPLES, alias="chain-1"
+    )
+    label_asym_id_1: str = Field(
+        ..., description="", examples=[], alias="label_asym_id-1"
```

Comment:
Do the empty `description`s in this file need to get one of the labels from `pisa_utils/models/labels.py`?